File: Basics of Computing and Programming/counter.py
def main():
    print("Please enter the number of coins:")
    quarters = int(input("# of quarters: "))
    dimes = int(input("# of dimes: "))
    nickels = int(input("# of nickels: "))
    pennies = int(input("# of pennies: "))
    total = 25 * quarters + 10 * dimes + 5 * nickels + pennies
    print(f"The total is {total // 100} dollars and {total % 100} cents")

if __name__ == "__main__":
    main()

File: Introduction to Programming in C++/Week3/counter.cpp
#include<bits/stdc++.h>
using namespace std;
int main() {

    // Input the number of coins
    int quarters;
    int dimes;
    int nickels;
    int pennies;
    cout << "Please enter the number of coins:" << endl;
    cout << "# of quarters:";
    cin >> quarters;
    cout << "# of dimes:";
    cin >> dimes;
    cout << "# of nickels:";
    cin >> nickels;
    cout << "# of pennies:";
    cin >> pennies;

    // Calculate the value
    int dollars = int(quarters * 0.25 + dimes * 0.1 + nickels * 0.05 + pennies * 0.01);
    int cents = quarters * 25 + dimes * 10 + nickels * 5 + pennies; 
    cents %= 100;

    // Output the ans
    cout << "The total is "<< dollars << " dollars and "<< cents <<" cents" << endl;
    return 0;
}